htdocs/admin/sms.php

```diff
@@ -124,10 +124,12 @@
 		if ($result)
 		{
 			setEventMessages($langs->trans("SmsSuccessfulySent",$smsfrom,$sendto), null, 'mesgs');
+			setEventMessages($smsfile->error, $smsfile->errors, 'mesgs');
```

Comment:
I want to be able able to set success message with the module like "Il reste xxx crédits sms" in case of success